tests/test-addons.py

```diff
@@ -142,10 +141,13 @@ def test_cilium(self):
         Sets up and validates Cilium.
         """
         print("Enabling Cilium")
-        p = Popen(
-            "/snap/bin/microk8s.enable cilium".split(), stdout=PIPE, stdin=PIPE, stderr=STDOUT
+        run(
```

Comment:
Switching these two `Popen` calls to `run` seem equivalent, might someone else might want to double check on that.